queries.sql

```diff
@@ -44,6 +44,13 @@ SELECT Species, MIN(weight_kg) FROM animals GROUP BY species;
 
 SELECT Species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY Species;
 
+SELECT full_name, name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE full_name = 'Melody Pond';
+SELECT species.name, animals.name FROM animals JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+SELECT FULL_NAME,NAME FROM ANIMALS FULL JOIN OWNERS ON ANIMALS.OWNER_ID = OWNERS.ID;   
+SELECT species.name, COUNT(species.name) FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.name;
+SELECT full_name,species.name, animals.name FROM animals JOIN species ON animals.species_id = species.id JOIN owners ON animals.owner_id = owners.id WHERE species.name = 'Digimon' AND full_name =  'Jennifer Orwell';
+SELECT full_name,COUNT(animals.owner_id) FROM animals JOIN owners ON animals.owner_id = owners.id GROUP BY full_name, animals.owner_id ORDER BY COUNT(animals.owner_id) DESC LIMIT 1;
+
```

Comment:
- [ ] Nice work writing queries that return accurate results. There is just one more thing you forgot to do, you didn't write a query for the requirement below:

![image](https://user-images.githubusercontent.com/77106962/207698756-f48abe8c-ff59-474a-b26f-7d07a2885e90.png)

Kindly write the query and update your snapshots with the correct result. 👍